app/src/main/java/io/github/droidkaigi/confsched2018/presentation/detail/SessionDetailFragment.kt

```diff
@@ -98,6 +108,43 @@ class SessionDetailFragment : Fragment(), Injectable {
         }
     }
 
+    /**
+     * Generate a shared text
+     *
+     * e.g.
+     * ConstraintLayout, now and future (Nicolas Roard, John Hoford, thagikura) #droidkaigi_room6
+     * https://droidkaigi.jp/2018/timetable/?session=16988
+     *
+     * @param session The [Session.SpeechSession] object
+     * @param sessionId The session ID
+     * @return A shared text
+     */
+    private fun generateSharedText(session: Session.SpeechSession, sessionId: String): String {
+        var speakerList = listOf<String>()
+        session.speakers.forEach { speakerList += it.name }
+        val speakers = speakerList.joinToString(", ")
```

Comment:
How about this? 👀 
session.speakers.map { it.name }.joinToString(", ")

Reply:
Thanks for suggestion. I'll simplify.